### PR TITLE
adding a couple operations and cleaning up a lot of descriptor.json

### PR DESCRIPTION
**Pull request by abramvandergeest**
yeah that

### CODE REVIEW

File: operations/image_processing/subSectionToImage/operation.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	ysub := a.params.LowerLeftCorner[1]
+
+	// a.logger.Debug("inputs", inputs)
+	a.logger.Info("Starting Operation subsectiontoimage.")
+	a.logger.Debugf("The subection is %dx%d in size with a lower left corner at (%d,%d)", wsub, hsub, xsub, ysub)
+
+	src := input.Img.(image.Image)
+
+	bnds := src.Bounds()
+	fmt.Println(bnds)
```
Comment (skothari-tibco) [inline operations/image_processing/subSectionToImage/operation.go]: Should this be in log or removed all-together?

File: operations/image_processing/subSectionToImage/operation.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	y1sub := ysub + hsub
+	if y1sub >= bnds.Max.Y {
+		y1sub = bnds.Max.Y
+	}
+	if x1sub >= bnds.Max.X {
+		x1sub = bnds.Max.X
+	}
+
+	newbnds := image.Rect(0, 0, x1sub-x0sub, y1sub-y0sub)
+	fmt.Println(newbnds)
```
Comment (skothari-tibco) [inline operations/image_processing/subSectionToImage/operation.go]: This one too.

File: operations/image_processing/subSectionToImage/operation.go [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+	err := metadata.MapToStruct(ctx.Params(), p, true)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &Operation{params: p, logger: ctx.Logger()}, nil
+}
+
+type initContextImpl struct {
```
Comment (skothari-tibco) [inline operations/image_processing/subSectionToImage/operation.go]: Are you using this here ?

File: operations/image_processing/subSectionToImage/operation.go (hunk 1 above)
Comment (abramvandergeest) [reply operations/image_processing/subSectionToImage/operation.go]: done

File: operations/image_processing/subSectionToImage/operation.go (hunk 3 above)
Comment (abramvandergeest) [reply operations/image_processing/subSectionToImage/operation.go]: done

File: operations/image_processing/subSectionToImage/operation.go (hunk 2 above)
Comment (abramvandergeest) [reply operations/image_processing/subSectionToImage/operation.go]: done

**Review by skothari-tibco — APPROVED** [review]

**Pull request merged**